common/src/main/kotlin/io/homeassistant/companion/android/database/AppDatabase.kt

```diff
@@ -125,7 +128,8 @@ import timber.log.Timber
         AutoMigration(from = 45, to = 46),
         AutoMigration(from = 46, to = 47),
         AutoMigration(from = 47, to = 48),
-        AutoMigration(from = 48, to = 49)
+        AutoMigration(from = 48, to = 49),
+        AutoMigration(from = 49, to = 50)
```

Comment:
```suggestion
        AutoMigration(from = 49, to = 50),
```
This has been allowed in a recent PR. It will make future changes better in Git to have a comma here.